sdk/src/utils.ts

```diff
@@ -271,3 +271,22 @@ export const getDeploymentData = async (
     throw new Error(`${contractName}.json or ${contractName}.wasm not found`)
   }
 }
+
+export const DENOMINATOR = BigInt(Math.pow(10, 12))
+
+export const calculatePriceImpact = (
+  startingSqrtPrice: SqrtPrice,
+  endingSqrtPrice: SqrtPrice
+): Percentage => {
+  const startingPrice = (startingSqrtPrice.v as bigint) * (startingSqrtPrice.v as bigint)
```

Comment:
Why here's casting to `bigint`? We pay lot of attention to current types to avoid thinking about it. If it's intentional redundancy, I'm not really supporter of this approach